api/src/main/java/org/openmrs/module/cohort/api/impl/CohortServiceImpl.java

```diff
@@ -94,9 +92,10 @@ public void voidCohort(CohortM cohort, String reason) {
 		}
 		cohort.setVoided(true);
 		cohort.setVoidReason(reason);
-
+		
 		for (CohortMember cohortMember : cohort.getCohortMembers()) {
-			Context.getService(CohortMemberService.class).voidCohortMember(cohortMember, "Cohort voided");
+			cohortMember.setVoided(true);
+			cohortMember.setVoidReason("Cohort voided");
```

Comment:
@ibacher I thought that, it would be better to call the two explicit would be better than calling `Context.getService` multiple times inside the `for loop` considering the performance, that is why I use `setVoided` and `setVoidReasin` methods.

Reply:
@ibacher After rename the `voidChort` method to `voidCohortM` ,no need to call `cohortMemberservice` void method.. when delete a `cohort` ,now automatically makes the `cohortMember` voided.